ui/src/app/components/resource-card.tsx

```diff
@@ -212,15 +209,11 @@ export class ResourceCard extends React.Component<Props, State> {
 
   async getDataSetByResourceId(id) {
     if (this.state.dataSetByResourceIdList.length === 0) {
-      try {
-        const dataSetList = await dataSetApi().getDataSetByResourceId(
-          this.props.resourceCard.workspaceNamespace,
-          this.props.resourceCard.workspaceFirecloudName,
-          this.resourceType, id);
-        return dataSetList.items;
-      } catch (ex) {
-        console.log(ex);
```

Comment:
I think it was intentional but I decided to change it because I think the behavior that results from NOT swallowing the error is more correct.

Effectively, swallowing the error would treat it as though the call succeeded with an empty list, which our UI treats as a sign that it is OK to delete.

Not swallowing the error will just cause the function to exit early and not attempt the delete.

Reply:
Makes sense - thank you for the explanation